charts/maildev/templates/smtp/service.yaml

```diff
@@ -11,8 +11,10 @@ spec:
       targetPort: smtp
       protocol: TCP
       name: smtp
-      {{ if and (eq .Values.services.smtp.type "NodePort")  .Values.services.smtp.nodePort }}
-      nodePort:  {{ .Values.services.smtp.nodePort }}
-      {{ end }}
+      {{- if (eq .Values.services.smtp.type "NodePort") }}
+      {{- if .Values.services.smtp.nodePort }}
```

Comment:
Why reconstruct this way? Is the `and` operator not working ? Just curious.

Reply:
I didn't confidently understand the AND operation. [Documentation Here](https://pkg.go.dev/text/template#hdr-Functions).

It says, 
> and computes the Boolean AND of its arguments, returning the first false argument it encounters, or the last argument.